core/src/main/java/com/klaytn/caver/tx/Account.java

```diff
@@ -49,4 +49,31 @@ public static Account create(Caver caver, KlayCredentials klayCredentials, int c
 
         return Account.create(caver, transactionManager);
     }
+
+    /**
+     * @deprecated  <p>In caver-java 1.0.0, we provided static methods to send transactions for `ValueTransfer`, `Account`, `Cancel`, and `SmartContract` classes. The static methods will be removed. </p>
+     *              <p>This deprecated method can be used only for Baobab Testnet</p>
+     *              use {@link #sendUpdateTransaction(AccountUpdateTransaction)} instead like this:
```

Comment:
```suggestion
     *              Use {@link #sendUpdateTransaction(AccountUpdateTransaction)} instead like this:
```

Reply:
Shouldn't we have more description after `like this:`? Or, you can remove that phrase simply.

Reply:
fixed